Sources/SwiftDocC/Model/Identifier.swift

```diff
@@ -495,22 +481,10 @@ public struct ResolvedTopicReference: Hashable, Codable, Equatable, CustomString
             self.absoluteString = self.url.absoluteString
         }
     }
-}
-
-typealias ResolvedTopicReferenceCacheKey = String
-
-extension ResolvedTopicReference {
-    /// Returns a unique cache ID for a pair of unresolved and parent references.
-    static func cacheIdentifier(_ reference: UnresolvedTopicReference, fromSymbolLink: Bool, in parent: ResolvedTopicReference?) -> ResolvedTopicReferenceCacheKey {
```

Comment:
You're simply removing dead code that isn't used anywhere?

Reply:
Yes. This was actually the code that I stumbled upon and made my first changes to but when nothing broke when I changed the return type I realized that I could simply remove it because nothing was using this.